e2e/content-services/permissions-component.e2e.ts

```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
         });
 
     });
+
+    describe('Role Consumer', function () {
+
+        beforeEach(async (done) => {
+
+            loginPage.loginToContentServicesUsingUserModel(fileOwnerUser);
+            contentServicesPage.goToDocumentList();
+
+            contentList.checkContentIsDisplayed(roleFolder.name);
+            contentServicesPage.checkSelectedSiteIsDisplayed('My files');
+            contentList.clickRowMenuActionsButton(roleFolder.name);
+            contentList.clickMenuActionNamed('PERMISSION');
+            permissionsPage.checkAddPermissionButtonIsDisplayed();
+            permissionsPage.clickAddPermissionButton();
+            permissionsPage.checkAddPermissionDialogIsDisplayed();
+            permissionsPage.checkSearchUserInputIsDisplayed();
+            permissionsPage.searchUserOrGroup(filePermissionUser.getId());
+            permissionsPage.clickUserOrGroup(filePermissionUser.getFirstName());
+            permissionsPage.checkUserOrGroupIsAdded(filePermissionUser.getId());
+            permissionsPage.clickRoleDropdown();
+            permissionsPage.selectOption('Consumer');
+            done();
+        });
+
+        it('[C274691] Should not to be able to Delete, see Info or Upload a file with Consumer permission', () => {
+
+            loginPage.loginToContentServicesUsingUserModel(filePermissionUser);
+            contentServicesPage.goToDocumentList();
+            browser.sleep(10000);
```

Comment:
can you remove this sleep and instead wait for a specific element? maybe waitForTableBody ?

Reply:
I couldn't find another workaround to do this, as it's dependent on the search indexing to finish beneath, I think.

Reply:
For indexing to finish you have a sleep above, at line 89.

Reply:
yes, tried using just that sleep, but the test fails without this sleep.